src/shared/Microsoft.Git.CredentialManager/Settings.cs

```diff
@@ -392,6 +398,30 @@ public bool IsCertificateVerificationEnabled
             }
         }
 
+        public string SslCaInfo
+        {
+            get
+            {
+                // First, verify that, if schannel is the HTTP backend, sslCaInfo must be explicitly enabled to be used
+                if (TryGetSetting(null, KnownGitCfg.Http.SectionName, KnownGitCfg.Http.SslBackend, out string backend) &&
+                    StringComparer.OrdinalIgnoreCase.Equals(backend, "schannel") &&
```

Comment:
Should these constants (`"schannel"` and `"openssl"`) be `const string`s in _Constants.cs_?

Reply:
Yes for `schannel`, but would you also want `openssl` included (even though it's not used anywhere here)?

Reply:
Because this setting is just a passthrough to libcurl, there are [more options](https://curl.se/libcurl/c/curl_global_sslset.html) than `schannel` and `openssl` (depending on platform). If `schannel` is the only one used here (so far, at least), I think I'd prefer to only include that for now (it keeps us from needing to keep up with mirroring all of libcurl's available options).

Reply:
If `openssl` is the default backend used by Git, would it be more appropriate to use a non-nullable `TlsBackend` for the setting, setting the value to `openssl` if it's unspecified in the user's config?